src/IntervalSets.jl

```diff
@@ -163,6 +150,32 @@ isequal(A::TypedEndpointsInterval, B::TypedEndpointsInterval) = isempty(A) & ise
 ==(A::TypedEndpointsInterval{L,R}, B::TypedEndpointsInterval{L,R}) where {L,R} = (leftendpoint(A) == leftendpoint(B) && rightendpoint(A) == rightendpoint(B)) || (isempty(A) && isempty(B))
 ==(A::TypedEndpointsInterval, B::TypedEndpointsInterval) = isempty(A) && isempty(B)
 
+function issubset(A::TypedEndpointsInterval, B::TypedEndpointsInterval)
+    Al, Ar = endpoints(A)
+    Bl, Br = endpoints(B)
+    return isempty(A) || ( Bl ≤ Al && Ar ≤ Br )
```

Comment:
```suggestion
    return isempty(A) | ( Bl ≤ Al && Ar ≤ Br )
```
(the branch in the original is probably actively harmful for performance)

Similar changes below.

Reply:
Thnaks for the suggestion!
I guess this code would be better, is it right?
```
return isempty(A) | ( Bl ≤ Al & Ar ≤ Br )
```